main.go

```diff
@@ -4,12 +4,14 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strconv"
```

Comment:
Suggestion: Consider moving the import of "strconv" closer to where it is used to improve readability.

Reply:
Thank you for your feedback! The AI gives your comment a score of 5